tests/serialization/as_dict_test.py

```diff
@@ -95,3 +99,63 @@ def test_dictify_with_only_includes(self, app):
         assert len(user_dict) == len(includes)
         assert set(user_dict.keys()) == set(includes)
         assert set(user_dict.values()) == set(map(lambda key: user_fields_dict[key], includes))
+
+    def test_dictify_with_sync_map(self, app):
```

Comment:
J'aurais bien aimé le parallèle de ce test avec async

Reply:
C est pas ce qu il y a em dessous?

Reply:
Si en effet, j'avais vu une boucle for je comprenais pas pourquoi.
Je m'attendais à avoir le même then dans les deux cas